src/tickets/tickets.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@nestjs/common';
+import { Transaction } from 'src/transactions/interfaces/transaction.interface';
+import { User } from 'src/users/interfaces/user.interface';
+import { PurchaseTicketDto } from './dto/purchase-ticket.dto';
+import { Ticket } from './interfaces/ticket.interface';
+import { TotpService } from './totp.service';
+
+@Injectable()
+export class TicketsService {
+  constructor(private readonly totpService: TotpService) {}
+  purchase(purchaseTicketDto: PurchaseTicketDto): Ticket {
+    const user: User = {
+      id: purchaseTicketDto.userId,
+      secret: 'WKSFANU2PIIOF5GK',
+      email: 'zerobasedteam@gmail.com',
+      name: 'zero-based',
+    };
```

Comment:
We should get the user from `UserService`, we already have a single declared user there.

Reply:
But in the future, we'll make a database call here to get the user directly as the `TicketService` does not depend on the `UserService`, what do you think?

Reply:
`UserService` should be the one doing the database calls on behalf of the `TicketService`.